### PR TITLE
Coverage any-beta should behave as without any coverage

### PR DESCRIPTION
**Pull request by kadhikari**
For more information: https://navitia.atlassian.net/browse/NAV-1482

[NAV-1482]: https://navitia.atlassian.net/browse/NAV-1482?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by SGrenet** [comment]
> Nom du coverage à rajouter : “any-beta”

isn't it "any_beta" in your PR?

**Review by xlqian — APPROVED** [review]

**Review by SGrenet — APPROVED** [review]

File: source/jormungandr/tests/places_tests.py [hunk 1]
```diff
@@ -363,6 +369,23 @@ def test_places_has_no_distance(self):
         assert len(places) == 1
         assert 'distance' not in places[0]
 
+    def test_places_to_valide_any_beta(self):
+        """A request with coverage=any_beta behaves as a request without any coverage"""
```
Comment (woshilapin) [inline source/jormungandr/tests/places_tests.py]:
```suggestion
        """A request with coverage=any-beta behaves as a request without any coverage"""
```

File: source/jormungandr/tests/places_tests.py [hunk 2]
```diff
@@ -363,6 +369,23 @@ def test_places_has_no_distance(self):
         assert len(places) == 1
         assert 'distance' not in places[0]
 
+    def test_places_to_valide_any_beta(self):
```
Comment (woshilapin) [inline source/jormungandr/tests/places_tests.py]: Not sure between
```suggestion
    def test_places_to_valid_any_beta(self):
```
or
```suggestion
    def test_places_to_validate_any_beta(self):
```

**Review by woshilapin — APPROVED** [review]

File: source/jormungandr/tests/places_tests.py (hunk 2 above)
Comment (kadhikari) [reply source/jormungandr/tests/places_tests.py]: :+1:

File: source/jormungandr/tests/places_tests.py (hunk 1 above)
Comment (kadhikari) [reply source/jormungandr/tests/places_tests.py]: :+1:

**Review by woshilapin — APPROVED** [review]

**Review by azime — APPROVED** [review]

**Review by wassimbenaissa — APPROVED** [review]

**Pull request merged**